docs/5.x upgrade guide.md

```diff
@@ -133,6 +129,13 @@ function YourWidget(props: WidgetProps) {
 }
 ```
 
+#### Deprecations
+
+##### Non-standard `enumNames` property
+
+`enumNames` is a non-standard JSON Schema field that RJSF supports prior to version 5 to apply labels that differed from an enumeration value. This behavior can still be accomplished with `oneOf` or `anyOf` containing `const` values, so `enumNames` may be removed from a future major version of RJSF. For more information, see [#532](https://github.com/rjsf-team/react-jsonschema-form/issues/532).
```

Comment:
consider changing `prior to version 5` to `and was deprecated in version 5` at the end of the sentence?